torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1276,6 +1276,11 @@ def make_input():
                 "test_fn_fwgrad_bwgrad",
                 dtypes=(torch.complex128,),
             ),
+            DecorateInfo(
+                "TestGradients",
+                "test_fn_gradgrad",
```

Comment:
Don't you want to limit tolerance reduction to just cuda tests?
```suggestion
                "test_fn_gradgrad",
                device="cuda",
```